Tests/YBottomSheetTests/BottomSheetControllerTests.swift

```diff
@@ -322,6 +289,56 @@ final class BottomSheetControllerTests: XCTestCase {
         let sut = makeSUT(viewController: UINavigationController(rootViewController: UIViewController()))
         XCTAssertFalse(sut.hasHeader)
     }
+
+    func test_onDimmer() {
+        let sut = SpyBottomSheetController(title: "", childView: UIView())
+        
+        XCTAssertFalse(sut.onDimmerTapped)
+        XCTAssertFalse(sut.isDismissed)
+
+        sut.simulateOnDimmerTap()
+        
+        XCTAssertTrue(sut.onDimmerTapped)
+        XCTAssertTrue(sut.isDismissed)
+    }
+    
+    func test_onSwipeDown() {
+        let sut = SpyBottomSheetController(title: "", childView: UIView())
+        
+        XCTAssertFalse(sut.onSwipeDown)
+        XCTAssertFalse(sut.isDismissed)
+
+        sut.simulateOnSwipeDown()
+        
+        XCTAssertTrue(sut.onSwipeDown)
+        XCTAssertTrue(sut.isDismissed)
+    }
+    
+    func test_onDidDismiss() {
+        let sut = SpyBottomSheetController(title: "", childView: UIView())
+        
+        XCTAssertFalse(sut.isDismissed)
+        
+        sut.simulateDismiss()
+        
+        XCTAssertTrue(sut.isDismissed)
+    }
+    
+    func test_forbidDismiss() {
+        let sut = SpyBottomSheetController(title: "", childView: UIView())
+        sut.appearance.isDismissAllowed = false
+        
+        XCTAssertFalse(sut.onSwipeDown)
+        XCTAssertFalse(sut.onDimmerTapped)
+        XCTAssertFalse(sut.isDismissed)
+        
+        sut.simulateOnDimmerTap()
+        sut.simulateOnSwipeDown()
```

Comment:
```suggestion
        sut.simulateOnDimmerTap()
        sut.simulateOnSwipeDown()
        _ = sut.accessibilityPerformEscape()
```
let's add in the accessibility escape gesture too.